core/controllers/reader_test.py

```diff
@@ -199,55 +148,6 @@ def test_give_feedback_handler(self):
         self.logout()
 
 
-class ExplorationStateClassifierMappingTests(test_utils.GenericTestBase):
```

Comment:
Keep this. It is part of the new framework.

Reply:
done.